testdata/configuration.ini

```diff
@@ -0,0 +1,12 @@
+[ovh]
+endpoint=ovh-eu
+application_key=TDPKJdwZwAQPwKX2
+application_secret=9ufkBmLaTQ9nz5yMUlg79taH0GNnzDjk
+consumer_key=5mBuy6SUQcRw2ZUxg0cG68BoDKpED4KY
+
+[swift]
```

Comment:
If you change this section's name following my previous comment on it, don't forget to update it here as well

Reply:
see comment above

Reply:
Understood